Find a functionally equivalent code:android.graphics.Canvas.drawPaint:COMMENT
Method Modifier: public      
Comment:/**
 * Fill the entire canvas' bitmap (restricted to the current clip) with
 * the specified paint. This is equivalent (but faster) to drawing an
 * infinitely large rectangle with the specified paint.
 *
 * @param paint The paint used to draw onto the canvas
 */

@@ -1,3 +1,3 @@
 {
-    native_drawPaint(mNativeCanvas, paint.mNativePaint);
+    native_drawPaint(mNativeCanvasWrapper, paint.mNativePaint);
 }

